apps/worker/src/routes/chat.ts

```
@@ -0,0 +1,417 @@
+const chatRouter = new Hono<{ Bindings: Env }>();
+
+// Schema for chat request
+const chatRequestSchema = z.object({
+  message: z.string().min(1).max(2000),
+  context_limit: z.number().int().min(1).max(10).optional().default(5),
+  stream: z.boolean().optional().default(false),
+  conversation_id: z.string().optional(),
+});
+
+// Schema for chat messages stored in DB
+const chatMessageSchema = z.object({
```

ESLint

[error] 'chatMessageSchema' is assigned a value but never used.

(no-unused-vars)